src/CMakeLists.txt

```diff
@@ -53,7 +53,6 @@ target_link_libraries(
                        raft 
                        ${CMAKE_SCOTCH_LIBS}
                        ${CMAKE_QTHREAD_LIBS} 
-                       affinity
```

Comment:
This does not look right to me. Will result in an linker error while compiling raftlib

Reply:
I think we can add that back in, already made that change. Now there's a but in the namespace injection code.....sub libraries are not getting the namespace with the current scope, working on a fix.